js/popup.js

```diff
@@ -0,0 +1,304 @@
+    list: [
+      {
+        text: 'See Live',
+        icon: './assets/icons/Icon-Export@2x.svg',
+      },
+      {
+        text: 'See Source',
+        icon: '.assets/icons/GitHub.svg',
+      },
+    ],
+  },
+  {
+    id: 'project_6',
+    images: {
+      img: './assets/Snapshoot-Portfolio-mobile6@2x.png',
+      cancelPop: './assets/icons/Disabled@2x.png',
+      altText: 'project 6 preview image',
+    },
+    name: 'Multi-Post Stories Gain+Glory',
+    heading: 'Keeping track of hundreds of components',
+    languages: [
+      'Ruby on Rails',
+      'Css',
+      'JavaScript',
+      'html',
+    ],
+    technologies: [
+      'Codekit',
+      'GitHub',
+      'Javascript',
+      'Bootstrap',
+      'Terminal',
+      'Codepen',
+    ],
+    // button: 'See Project',
+    description: 'Project_6 Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry\'s standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the releaLorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry\'s standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the relea',
+    list: [
+      {
+        text: 'See Live',
+        icon: './assets/icons/Icon-Export@2x.svg',
+      },
+      {
+        text: 'See Source',
+        icon: './assets/icons/GitHub.svg',
+      },
+    ],
+  },
+];
+
+function createCard(cardObject) {
+  const li = document.createElement('li');
+  li.className = 'card--section';
+  li.innerHTML = `<div class="image--container">
+  <img src="${cardObject.images.img}" alt="${cardObject.images.altText}">
+  </div>
+  <div class="btn--container">
+    <h4>${cardObject.name}</h4>
+    <ul class="ul--section">
+        ${cardObject.languages.map((lang) => `<button type="button" class="btn">
+        ${lang}</button>`).join('')}
+    </ul>
+  </div>
+  <div class="see--project">
+    <button type="button" class="btn--style ${cardObject.id}">See Project</button>
+  </div>`;
+
+  return li;
+}
+
+projects.forEach((project) => {
+  const card = createCard(project);
+  mainWorkSection.appendChild(card);
+});
+
+function createPopup(object) {
+  const popupCard = document.createElement('div');
+  popupCard.classList.add('card-popup');
+  popupCard.id = object.id;
+  popupCard.innerHTML = `
+  <div class="project-img">
+    <img class="card-img" src="${object.images.img}" alt="${object.images.altText}">
+    <img class="hide-icon" src="./assets/icons/Disabled@2x.png"></div>
```

Comment:
- [ ] kindly make sure to have elements aligned as shown in the design, currently, the close button for the pop-up window appears to be misaligned as shown below:
#### Your project
![image](https://user-images.githubusercontent.com/61377322/146307842-6f8140f6-d798-4ebd-b781-410617a32cd9.png)
#### Figma design
![image](https://user-images.githubusercontent.com/61377322/146307897-f9b7a44e-10c3-453f-a7de-a67cd7256b1a.png)